main.go

```diff
@@ -50,34 +51,38 @@ func main() {
 watchloop:
 	for evt := range watch.ResultChan() {
 		pod := evt.Object.(*v1.Pod)
-		switch evt.Type {
-		case "ADDED", "MODIFIED":
-			for _, status := range pod.Status.ContainerStatuses {
-				if w := status.State.Waiting; w != nil {
-					if w.Reason == "CrashLoopBackOff" {
-						log.Printf("Pod %v/%v is in CrashLoopBackOff, deleting", pod.Namespace, pod.Name)
-						for {
-							if _, _, err = chat.PostMessage(*channelId, slack.MsgOptionText(
-								fmt.Sprintf("Deleting crashlooping pod `%s/%s`", pod.Namespace, pod.Name), false)); err != nil {
-								var rlError *slack.RateLimitedError
-								if errors.As(err, &rlError) {
-									log.Printf("Rate limited, sleeping %s", rlError.RetryAfter)
-									time.Sleep(rlError.RetryAfter)
+		if v, found := pod.Annotations["crashloopbackon"]; found {
```

Comment:
If we do a label instead of an annotation, you can specify it as a filter in the ListOptions above...